testsmanual/BenchmarkDotNet.IntegrationTests.ClassicNetFramework/BenchmarkDotNet.IntegrationTests.ClassicNetFramework.csproj

```diff
@@ -0,0 +1,161 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
using the old project files will be expensive to maintain

Reply:
Agreed. Should I just remove that test then? I guess it's not so important nowadays.